src/models/User.ts

```diff
@@ -3,4 +3,5 @@ import { IUser } from '@hunteroi/discord-verification';
 export interface User extends IUser {
     createdAt: Date;
     updatedAt: Date;
+    isDeleted?: Date | null;
```

Comment:
Because the optional operator and the type of the value are two different things. The type of `isDeleted` is `Date | null | undefined` but the optional operator `?` allows to bypass some restrictions like when you want to create a user:
```ts
// without the ?
interface Sandwich {
  name: string | null;
  price: number;
}
const dago: Sandwich = { name: null, price: 5.0 };

// with the ?
interface SodaBottle {
  size: number;
  color?: Color | null;
}
const coca: SodaBottle = { size: 25 };
```

This is all due to transpilation checks...